### PR TITLE
OSSM-6010 [DOC] Update user-workload monitoring so Kiali can fetch metrics from Thanos

### PR DESCRIPTION
**Pull request by gwynnemonahan**
[OSSM-6010](https://issues.redhat.com//browse/OSSM-6010) [DOC] Update user-workload monitoring so Kiali can fetch metrics from Thanos

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSSM-6010

Link to docs preview:
https://72324--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-observability#ossm-integrating-with-user-workload-monitoring_observability 

QE review:
- [x] QE has approved this change.

Additional information:
Updating Step 1 due to changes in OpenShift 4.15. 

Also changed "may" to "might" in NOTE to be in line with current IBM/Red Hat Style. 

Per Jacek, change made to Step 2 per his PR https://github.com/maistra/istio-operator/pull/1653 . This is not part of the Jira issue associated with this PR, however, the change is necessary and in the same file so it seems sensible to make the change before this PR has gone through all approval processes.

### CODE REVIEW

**Review by jewertow — COMMENTED** [review]
LGTM

**Comment by gwynnemonahan** [comment]
Per Jacek, change made to Step 2 per his PR https://github.com/maistra/istio-operator/pull/1653 if using Istio Operator 2.4. It has been added as an bullet point under Step 2 for configuring Kiali.

**Review by prachiyadav — APPROVED** [review]

**Review by kowen-rh — COMMENTED** [review]
This mostly looks good to me, I just have a couple of small comments and one question. :)

/remove-label peer-review-in-progress
/label peer-review-done

File: modules/ossm-integrating-with-user-workload-monitoring.adoc [hunk 1]
```diff
@@ -19,31 +19,47 @@ The following steps show how to integrate Service Mesh with user-workload monito
...
+        use_kiali_token: false
+      query_scope:
+        mesh_id: "basic-istio-system"
+      thanos_proxy:
+        enabled: true
+      url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
 ----
-
-. Configure Kiali for user-workload monitoring:
+** If you use Istio Operator 2.4, use this configuration to configure Kiali for user-workload monitoring:
```
[the remark this answers is not in the document]
Comment (gwynnemonahan) [reply modules/ossm-integrating-with-user-workload-monitoring.adoc:62]: The change in 4.15 necessitates this specific configuration only for Istio Operator 2.4. A number of users on Istio Operator 2.4 have reported issues, and this is a fix for those, and is specific only to Istio Operator 2.4.

**Pull request merged**